text/0000-localizing-api-messages.md

```diff
@@ -0,0 +1,135 @@
+
+- Start Date: 2022-03-29
+- RFC PR: 
+- FOLIO Issue: 
+- Submitter : Radhakrishnan Gopalakrishnan (rgopalakrishnan@ebsco.com)
+- Co-Submitter(s) : Zak Burke (zburke@ebsco.com)
+- Sub Group :
+
+# Localizing API (Backend) Messages
+
+## Summary
+  The purpose of this RFC is to define an approach that allows API developer to provide
+messages in the API response depending on the language preference set in the request. 
+
+
+## Motivation
+
+- Help users understand the information coming from the application backend using a language they understand better
+- To allow FOLIO platform to be used by users across the world 
+- To create a tremendous opportunity for growth that we can never could have achieved within just one country.
+
+### Terminology
+- Static Message - E.g. _This is something you need translate_
+- Static Message with placeholder - _You can ONLY renew {n} times_
```

Comment:
Is the name field of a reference data record a static message?

Reply:
From what I understand, values in the database are referred to as controlled vocabulary. So, the name column value from a DB record falls into that category

Reply:
> From what I understand, values in the database are referred to as controlled vocabulary. 

Please be careful using the term Controlled Vocabulary. It has a specific meaning within the library domain (to do with how change management for those values work) 

Not all reference records are controlled vocabulary.

> So, the name column value from a DB record falls into that category

Which category are you referring to?

As I understand it, the names of reference records are considered `dynamic` values. @zburke I think you coined that terminology originally, how wrong am I?

Reply:
If controlled vocabulary has no relevance when it comes to localization, I will replace it with "Dynamic messages", to refer to the data in the DB that needs to be localized.

Reply:
> If controlled vocabulary has no relevance when it comes to localization, I will replace it with "Dynamic messages", to refer to the data in the DB that needs to be localized.

I think replacing it with dynamic values is a good idea.